dev-client/src/screens/SiteNotesScreen/ReadPinnedNoteScreen.tsx

```diff
@@ -37,33 +42,51 @@ type Props = {
 export const ReadPinnedNoteScreen = ({projectId}: Props) => {
   const {t} = useTranslation();
   const navigation = useNavigation();
+  const syncNotifications = useSyncNotificationContext();
+
   const project = useSelector(selectProject(projectId));
   const content = project?.siteInstructions;
 
   const handleClose = () => {
     navigation.pop();
   };
 
+  const handleMissingProject = useHandleMissingSiteOrProject();
+  const handleMissingPinnedNote = useCallback(() => {
```

Comment:
Suggestion: turn this into a reusable handler hook like `useHandleMissingSiteOrProject()`